iphone/Maps/Categories/UIStackView+ArrangedSubviewWithSeparator.swift

```diff
@@ -0,0 +1,103 @@
+extension UIStackView {
```

Comment:
I'm curious, is it the simplest method to fix the issue?

Reply:
The problem was not easy to fix because each view controller in PlacePageInfo contains own separator on bottom with constraints (it is not very comfortable to work with):
<img width="1728" alt="SCR-20231005-lute" src="https://github.com/organicmaps/organicmaps/assets/79797627/c08ed9ee-139a-45b1-8bd8-baa4825eb095">

So there are some approaches to fix that that i found:
1. hardcode each view when it is adding to the stack and try to inspect is it have separator or not, cast it and change it's visibility (a lot of hardcoding)
2. to rewrite all screen with UITableView (very hard)
3. to move responsibility of adding separators from each cells to stack view, remove separators from cells at all and use "tableview's style approach" for stack view then you just setup separator at one place
```swift
let tableView = UITableView()
tableView.separatorInset = UIEdgeInsets.zero
```
So i choose 3rd one because it is a bad idea to add separator to each view manually in storyboard and constraint them to some views. It is also not responsibility of cell to add and layout separators because cell shouldn't know its position in collection/stack. 
Now if we want to add some new info or rearrange PlacePage developer should not think about adding of separators manually and layouting them. And style of separator can be changed from **one place** in some clicks, but not in every view controller as before.
---
PS: I can move this code to private extension to PlacePage VC but i think that it will be Ok to use same approch if needed in some other places where we use UIStackView instead of UITableView.

Reply:
Thanks for the explanation! Why is the table view approach more complex than the stack view one?

P.S. The KISS principle suggests avoiding doing unnecessary work if it's not needed ) If it's used now only in PP, let's move it there.

Reply:
TableView approach is easy to implement when building from scratch but in our case it needs to do some refactoring:
- changing layout and outlets in storyboard
- creating custom empty cells that will contain view controllers 
- registering/dequeuing of cells
- implement UITableViewDelegate and UITableViewDataSource protocols
- inserting resulting self-sizing non-scrollable UITableView inside of stackView

And we also needs to:
- remove separators from storyboard like i do
- set separators insets manually in uitableview

So it can be done but needs more time to refactor than just add separators manually